lua/refactoring/debug/debug_utils.lua

```diff
@@ -0,0 +1,22 @@
+local M = {}
+
+function M.get_code_gen(refactor, bufnr)
+    local code_gen = refactor.config:get_code_generation_for()
+    if not code_gen then
+        error(string.format("No code generator for %s", vim.bo[bufnr].ft))
+    end
+    return code_gen
+end
+
+function M.get_debug_path(refactor, point)
```

Comment:
provide node instead of point just because we may use other things?

Thoughts?

Reply:
I think this is fine for now (used by `printf` and `printVar`. If we need to change in the future we can.